data.sql

```diff
@@ -14,16 +14,16 @@ VALUES ('Pikachu', '2021-01-07', 15.04, false, 1),
 
 -- Insert data for Devimon
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
-VALUES ('Devimon', '2017-05-12', 11, true, 5),
+VALUES ('Devimon', '2017-05-12', 11, true, 5);
 
--- Inserting new data after species column addition
 
--- Insert data for Charmander
+-- Adding new animals after the addition os species column
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species) VALUES
 ('Charmander', '2020-02-08', -11, false, 0, 'Lizard'),
 ('Plantmon', '2021-11-15', -5.7, true, 2, 'Dog'),
 ('Squirtle', '1993-04-02', -12.13, false, 3, 'Turtle'),
 ('Angemon', '2005-06-12', -45, true, 1, 'Cat'),
 ('Boarmon', '2005-06-07', 20.4, true, 7, 'Boar'),
-('Blossom', '1998-10-13', 17, true, 3, 'Cat'),
-('Ditto', '2022-05-14', 22, true, 4, 'Rabbit');
+('Blossom', '1998-10-13', 17, true, 3, 'Rabbit'),
+('Ditto', '2022-05-14', 22, true, 4, 'Cat');
```

Comment:
- [ ] 👍 Great job adding the data! However, it's important to ensure that the data aligns with the project requirements. I noticed that you added some values in the `species` column that were not specified in the project requirements. To maintain consistency with the project requirements, kindly review the data you've added and remove any species values that were randomly entered.
- [ ] Additionally, if you make any updates to the data, please include new queries and screenshots after the updates, if necessary. This will help ensure that your data is in accordance with the project's specifications. Keep up the good work! 👍📊📸